### PR TITLE
libraw 0.19.2

### PR DESCRIPTION
**Pull request by niheaven**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Moisan** [comment]
Thank you for your first pull-request! `rawtoaces` needs to have its revision bumped. Could you bump (+1) the revision of `rawtoaces` and commit with the message `rawtoaces: revision bump for libraw`?

**Comment by niheaven** [comment]
Bumped revision of rawtoaces #36026

**Comment by niheaven** [comment]
- Add OpenMP support following #35637's method
- Bump some formula's revision who use libraw as dependency

**Comment by Moisan** [comment]
`brew audit` fails:
```
        Error: 2 problems in 1 formula detected
libraw:
  * C: 17: col 3: dependency "jasper" (line 17) should be put before dependency "libomp" (line 16)
  * C: 18: col 3: dependency "jpeg" (line 18) should be put before dependency "libomp" (line 16)
```
Please reorder the dependencies accordingly.

**Comment by niheaven** [comment]
Sorry for my carelessness, since the first commit is submit by `brew bump-formula-pr`, and after adding OpenMP, I just install it to ensure it works while ignoring check `brew audit` and `brew test` result.

I've check all 5 formulaes, and now they're okay.

File: Formula/libraw.rb [hunk 1]
```diff
@@ -24,7 +25,10 @@ class Libraw < Formula
 
   def install
     system "./configure", "--prefix=#{prefix}",
-                          "--disable-dependency-tracking"
+                          "--disable-dependency-tracking",
+                          "ac_cv_prog_c_openmp=-Xpreprocessor\\ -fopenmp",
```
Comment (fxcoudert) [inline Formula/libraw.rb]: - The `\\` is, I think, unnecessary.
- Do we really need OpenMP support? It adds one more dependency

**Pull request closed without merging**